### ISSUE TITLE
Singals are ignored, breaking monitoring systems

### ISSUE THREAD

**Issue by sbienkow** [osixia/docker-keepalived#50]
Monitoring exporters, like this one: <https://github.com/cafebazaar/keepalived-exporter> rely on being able to send signal to docker container in order for keepalived to dump its stats/data files.

Because in this image the PID1 is merely a startup process (it doesn't `exec keepalived`), it is the one that receives the signals. It ignores those signals completely, causing it to fail. I believe fixing it might require allowing containers to register custom signal handlers here: https://github.com/osixia/docker-light-baseimage/blob/master/image/tool/run#L892-L894

**TL;DR**:
I'd expect running `docker kill -s 12 keepalived` to cause it to create `/tmp/keepalived.stats`.

**Comment by BertrandGouny** [comment]
Thanks for reporting this! Due to the recent rewrite, this issue is no longer applicable. Feel free to reopen or open a new issue if it still occurs with the new version.
